release-announcements/20210701-yt-4.0.0.txt

```diff
@@ -0,0 +1,82 @@
+The yt community is proud to announce the release of yt-4.0!
+
+yt (http://yt-project.org) is an open source, community-developed toolkit for analysis and visualization of volumetric data.
+
+If you are interested in contributing to yt, please see our repository on GitHub at http://github.com/yt-project/yt
```

Comment:
Is this being sent out to users and dev? maybe these lines belong at the end then since they'll know what yt is?

Reply:
Usually it gets forwarded along to other mailing lists.  I can move this to the bottom, however.